### PR TITLE
[IE][VPU] WIP : Add new operation Interpolate-4 to IE MyriadX plugin

### PR DESCRIPTION
**Pull request by DariaMityagina**

### CODE REVIEW

File: inference-engine/tests_deprecated/functional/vpu/common/layers/myriad_layers_interpolate_test.hpp [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+
+    std::map<std::string, std::string> params;
+    params["antialias"] = std::to_string((int)antialias);
+    params["sampleNearestMode"] = std::to_string(sampleNearestMode);
+    params["sampleShapeCalcMode"] = std::to_string(sampleShapeCalcMode);
+    params["sampleCoordTransMode"] = std::to_string(sampleCoordTransMode);
+    params["axis"] = std::to_string(axis);
+    params["scales"] = std::to_string(scales);
+
+    // ASSERT_NO_FATAL_FAILURE(makeSingleLayerNetwork(LayerInitParams("Interpolate").params(params),
```
Comment (andrejsokolov) [inline inference-engine/tests_deprecated/functional/vpu/common/layers/myriad_layers_interpolate_test.hpp:321]: @DariaMityagina, please remove dead code

File: inference-engine/tests/functional/plugin/myriad/common/myriad_layers_interpolate_test.hpp [hunk 2]
```diff
@@ -0,0 +1,220 @@
+// // Copyright (C) 2020 Intel Corporation
```
Comment (andrejsokolov) [inline inference-engine/tests/functional/plugin/myriad/common/myriad_layers_interpolate_test.hpp:1]: what is this file for?

File: inference-engine/tests/functional/plugin/myriad/common/myriad_layers_interpolate_test.cpp [hunk 3]
```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "myriad_layers_interpolate_test.hpp"
+
+// INSTANTIATE_TEST_CASE_P(
```
Comment (andrejsokolov) [inline inference-engine/tests/functional/plugin/myriad/common/myriad_layers_interpolate_test.cpp:7]: why is it commented?

Comment (DariaMityagina) [reply inference-engine/tests/functional/plugin/myriad/common/myriad_layers_interpolate_test.cpp:7]: It is not ready yet, I am currently working on tests.

File: inference-engine/src/legacy_api/include/legacy/ie_layers.h [hunk 4]
```diff
@@ -1536,6 +1536,76 @@ class INFERENCE_ENGINE_INTERNAL_CNNLAYER_CLASS(PadLayer): public CNNLayer {
     ~PadLayer() override;
 };
 
+/**
+ * @deprecated Migrate to IR v10 and work with ngraph::Function directly. The method will be removed in 2021.1
```
Comment (taka-no-me) [inline inference-engine/src/legacy_api/include/legacy/ie_layers.h:1540]: @andrejsokolov @DariaMityagina 2021.1 is out already. Please implement interpolate without adding new deprecated class

**Review by taka-no-me — CHANGES_REQUESTED** [review]

File: inference-engine/src/legacy_api/include/legacy/ie_layers.h (hunk 4 above)
Comment (DariaMityagina) [reply inference-engine/src/legacy_api/include/legacy/ie_layers.h:1540]: Okay, thanks.

**Pull request closed without merging**